tests/features/content_types/blog_post.feature

```diff
@@ -17,13 +21,61 @@ Feature: Blog Content type
       | :lng                                | -122.238717     |
       | field_location_phone                | +1234567890     |
 
+  @wysiwyg
   Scenario: Create basic blog and check fields
-    Given I am logged in as a user with the Administrator role
     When I go to "/node/add/blog"
-    And I fill in "Title" with "From OpenY Automation Blogger"
+    And I fill in "Title" with "Behat test blog"
     And I select "BEHAT BRANCH 01" from "Location"
-    And I fill in the following:
-      | Category    | BEHAT CATEGORY ONE                          |
-      | Description | This could be a draft for a wonderful post. |
+    And I fill in "Category" with "BEHAT CATEGORY ONE"
+    When I click "//*[@id='edit-group-content-area']/summary" xpath element
+    And I scroll to "#cke_edit-field-blog-description-0-value" element
+    And I should see an "#cke_edit-field-blog-description-0-value" element
```

Comment:
Seems we don't need that line. Cause it is always true, if previous one is done. Am I correct?